examples/hyper.rs

```diff
@@ -1,52 +1,54 @@
-use failure::{Compat, Fail};
-use futures::prelude::*;
-use hyper::{
-    client::connect::{Connect, Connected, Destination},
-    Client, Uri,
-};
-use std::io::{prelude::*, stdout};
-use std::net::SocketAddr;
-use tokio_socks::{tcp::Socks5Stream, Error};
-use tokio_tcp::TcpStream;
+//use failure::{Compat, Fail};
```

Comment:
I think you can just remove this file for now. Hyper examples can be added later.

Reply:
Perfect - thanks